apps/marketplace/components/BuyerSpecialist/BuyerSpecialistSelectStage.js

```diff
@@ -109,83 +107,82 @@ export class BuyerSpecialistSelectStage extends Component {
             atLeastOneSeller: 'You must add at least one seller'
           }}
         />
-        <PanelCategorySelect
-          id="select-seller"
-          categories={categories}
-          onChange={e => this.handleSellerCategorySelect(e.target.value)}
-          selectedCategory={this.props[this.props.model].sellerCategory}
-          label="Category"
-        />
-        {this.props[this.props.model].sellerCategory && (
+        <div>
```

Comment:
Is this extra `div` necessary?

Reply:
It doesn't seem to be, at least not anymore.